### PR TITLE
Add folder attribute to the page class

### PR DESCRIPTION
**Pull request by blackdog308**
This goal of this pull request is to add a folder attribute to a page object to show what folder (relative to the root) the page is in.

Implementation:
Take rel_path from _walker function and pass through to the page object via _load_file & _parse

### CODE REVIEW

**Comment by padraic-padraic** [comment]
Happy to merge this. If you could just sync with master and rerun the tests (I think they were failing due to codehilite changes, not your work, but still) that would be much appreciated

**Comment by blackdog308** [comment]
> Happy to merge this. If you could just sync with master and rerun the tests (I think they were failing due to codehilite changes, not your work, but still) that would be much appreciated

Still getting issues with the checks:

`ERROR: InvocationError for command /home/travis/build/Flask-FlatPages/Flask-FlatPages/.tox/py38/bin/flake8 --statistics flask_flatpages/ --import-order-style smarkets (exited with code 1)`

File: flask_flatpages/flatpages.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 flask_flatpages.flatpages
 =========================
 
-Flatpages extension.
+Flatpages extension.
```
Comment (padraic-padraic) [inline flask_flatpages/flatpages.py:6]: Here's the issue. It afailed our (very strict!) linting due to trailing whitespace. You can run the tests locally to check things like this before commiting.

**Review by padraic-padraic — CHANGES_REQUESTED** [review]

File: flask_flatpages/flatpages.py (hunk 1 above)
Comment (blackdog308) [reply flask_flatpages/flatpages.py]: Merged in updated - sorry my first time using git to contribute so I'm still getting my head around how to use it.

**Pull request merged**